server/mail.md

```diff
@@ -1,69 +1,101 @@
+
+* Use SMTP plugins: You can install [SMTP plugins](https://wordpress.org/plugins/search/smtp/) from the WP.org plugin directory. These type of plugins overrides `wp_mail()` default settings to use PHPMailer or any other mailing library of their choice, which utlize remote third party mailing services to send mail on your behalf of your domain.
+
 ### Ensuring Deliverability
 
+Once we have our system sending mails, we need to ensure that they are actually delivered to the recipients. There are a few things we need to check:
+
 #### Ensure Proper Return Address is Used
 
-By default, the WordPress mailer fills in the `From:` field with *wordpress@example.com* and the `From:` name as *WordPress*.
+By default, the WordPress wrapper fills in the `From:` field with `wordpress@example.com` and the `From:` name as *WordPress*. This is fine if this is a valid e-mail address. For example, if your domain is `example.com`, your host shall pass `wordpress@example.com` on for delivery. It will probably send your mail as long as `example.com` is setup to send and receive mail, even if `wordpress` is not a valid mailbox. Some old mail delivery agents will check if the `From:` address is valid and can receive mails. This check is becoming less common nowadays, but we recommend to make sure to use a valid email address and the mailbox exists in your receiving mail server. If you are using the default `From:` address, simply verify that you can receive mails in the `wordpress@example.com` mailbox. Remember that `example.com` is the domain of your WordPress installation.
+
+But if you don't want to send the emails from the same domain as the one exhibited in your installation then you must introduce some code changes in your WordPress. For example, if your WP domain is `example.org` but you want to send the emails from `mail.example.org` the mail may not send because the DNS records of `example.org` might not authorize its mails to be handled by your mail server, because the default `From:` address will be `wordpress@wordpress.org`. To change the default `From:` address, your wordpress can [use the `wp_mail_from` filter](https://developer.wordpress.org/reference/hooks/wp_mail_from/). For the name, you can use [the `wp_mail_from_name` filter](https://developer.wordpress.org/reference/hooks/wp_mail_from_name/). 
+
+Also be aware that you can also force the `From:` address configuration in your configured MTA server. Refer to the related documentation for your MTA.
+
+#### The Reply-To Address
+
+The WordPress wrapper ignores the `Reply-To:`, so it will be coincidental with the `From:` address, following the example above, `wordpress@example.com` by default. If you want your users to be able to reply to your mails make sure that this mailbox exists in your receiving mail server. The default `Reply-To:` is always going to be the same as the `From:` address unless you explicitly set it to a different address in the mail headers. 
```

Comment:
```suggestion
The WordPress wrapper ignores the `Reply-To:`, so it will be coincidental with the `From:` address, following the example above, `wordpress@example.com` by default. If you want your users to be able to reply to sent emails, make sure that this mailbox exists on your receiving mail server. The default `Reply-To:` is always going to be the same as the `From:` address, unless you explicitly set it to a different address in the mail headers. 
```